Calculator/src/main/java/com/programmers/calculator/engine/io/Console.java

```diff
@@ -0,0 +1,40 @@
+package com.programmers.calculator.engine.io;
+
+import com.programmers.calculator.engine.repository.CalculatorRepository;
+import com.programmers.calculator.engine.repository.MemoryCalculatorRepository;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class Console implements Input, Output  {
+
+    private final Scanner scanner = new Scanner(System.in);
+    private CalculatorRepository repository = new MemoryCalculatorRepository();
+
+    @Override
+    public String optionInput()  {
+        System.out.println("1. 조회");
+        System.out.print("2. 계산\n\n");
+        System.out.print("선택 : ");
+        String str = scanner.nextLine();
+        System.out.println();
+        return str;
+    }
+
+    @Override
+    public String calculationInput() {
+        String str = scanner.nextLine();
+        return str;

```

Comment:
```suggestion
	    return scanner.nextLine();
```

값을 받아와서 또다른 로직을 통해 처리하는 것이 아니면 바로 반환해도 될 것 같네요.!